xdsl/transforms/shape_inference.py

```diff
@@ -34,3 +34,12 @@ class ShapeInferencePass(ModulePass):
 
     def apply(self, ctx: MLContext, op: builtin.ModuleOp) -> None:
         PatternRewriteWalker(ShapeInferenceRewritePattern()).rewrite_module(op)
+
+
+def ShapeInferencePassHelper(op: builtin.ModuleOp):
```

Comment:
Our coding style is to use snake_case for function names, in this case giving something like this:
```suggestion
def shape_inference_pass_helper(op: builtin.ModuleOp):
```
Could you please move this above the pass definition, and also call it from the pass itself?